src/main/java/org/jfrog/hudson/ArtifactoryBuilder.java

```diff
@@ -73,6 +76,11 @@
     public static final class DescriptorImpl extends Descriptor<GlobalConfiguration> {
 
         private boolean useCredentialsPlugin;
+        private List<JFrogPlatformInstance> jfrogInstances;
+        /**
+         * @deprecated: Use org.jfrog.hudson.ArtifactoryBuilder.DescriptorImpl#getJfrogInstances()
+         */
+        @Deprecated
         private List<ArtifactoryServer> artifactoryServers;
```

Comment:
Needs to be marked `transient` as described in [the Jenkins developer documentation](https://wiki.jenkins.io/display/JENKINS/Hint+on+retaining+backward+compatibility#Hintonretainingbackwardcompatibility-Scenario:Renameafield). That it was not broke JCasC export.